examples/pytorch/graphsage/advanced/train_sampling_unsupervised.py

```diff
@@ -133,10 +133,9 @@ def run(proc_id, n_gpus, args, devices, data):
         # blocks.
         tic_step = time.time()
         for step, (input_nodes, pos_graph, neg_graph, blocks) in enumerate(dataloader):
+            input_nodes = input_nodes.to(nfeat.device)
             batch_inputs = nfeat[input_nodes].to(device)
-            pos_graph = pos_graph.to(device)
-            neg_graph = neg_graph.to(device)
-            blocks = [block.int().to(device) for block in blocks]
+            blocks = [block.int() for block in blocks]
```

Comment:
Since `pos_graph`, `neg_graph`, `blocks` all reside in the same device as dataloader (L105), moving to `device` seems redundant.